### ISSUE TITLE
The currency symbol in Payments doesn't show the local currency symbol for non-US locales - reported by Mahendra Liya

### ISSUE THREAD

**Issue by mvtglobally** [Expensify/App#9820]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/docs/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed: 
Prerequisite: User located outside of US 
1. Go to chat and Request / Send Money.
2. Notice the currency symbol is as per the current locale.
3. Go to Settings and notice currency symbol in front of the balance.

## Expected Result:
The currency symbol under Settings -> Payments should shows the correct currency symbol as per the locale.

## Actual Result:
The currency symbol defaults to $

## Workaround:
un known 

## Platform:

Where is this issue occurring?

- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:** 1.1.79-0
**Reproducible in staging?:** Y 
**Reproducible in production?:** Y 
**Email or phone of affected tester (no customers):** 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation 
<img width="1920" alt="currency-symbol-bug" src="https://user-images.githubusercontent.com/43995119/178181069-237ce0ce-abac-4826-a598-6c19d270baf0.png">

https://user-images.githubusercontent.com/43995119/178181073-c56f013e-064c-43c3-8004-824f7e9fb2ae.mp4

**Expensify/Expensify Issue URL:** 
**Issue reported by:** @Mahendra Liya
**Slack conversation:** https://expensify.slack.com/archives/C01GTK53T8Q/p1656409899433359

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by parasharrajat** [comment]
This is expected behavior.

**Comment by Gonals** [comment]
I think it'd be nice to change that to the local currency, but not a huge priority. Setting it as monthly.

**Comment by liyamahendra** [comment]
@iwiznia The issue is - if you local currency isn't USD, notice the currency symbol in the Send Money / Request Money chats. 

![](https://user-images.githubusercontent.com/43995119/178181069-237ce0ce-abac-4826-a598-6c19d270baf0.png)

I believe the same symbol should reflect under Settings isn't it?

**Comment by iwiznia** [comment]
No, because the wallet is ALWAYS AND ONLY in USD, it does not change depending on your local currency

**Comment by liyamahendra** [comment]
Ah, okay @iwiznia! Do you think it makes sense to show the amount in USD in the chat and not the local currency then?
